Samples/Samples/ViewModel/ClipboardViewModel.cs

```diff
@@ -24,6 +25,21 @@ public string FieldValue
             set => SetProperty(ref fieldValue, value);
         }
 
+        public override void OnAppearing()
+        {
+            Clipboard.ClipboardContentChanged += OnClipboardContentChanged;
+        }
+
+        public override void OnDisappearing()
+        {
+            Clipboard.ClipboardContentChanged -= OnClipboardContentChanged;
+        }
+
+        async void OnClipboardContentChanged(object sender, EventArgs args)
+        {
+            await DisplayAlertAsync("Clipoard content changed!");
```

Comment:
Maybe not have an alert... A status label might be better - you could have a "Clipboard changed: {the text}" or something.

I wonder if there are native features to clear the clipboard for some reason? That might also be a nice feature. But, that can be a separate PR.

Reply:
Not all implementations include the changed content, and as we only support text clipboard content I would advise against including the changed content in the event. You can always grab clipboard content in the event handler, but this way we have the bare minimum api across all platforms without fetching the clipboard :)

Reply:
On, no. I was talking about instead of using `DisplayAlertAsync`, rather update some label in the sample UI. I want to move away from popups as they break UI tests and are not as nice as a status label.

Reply:
Done